### PR TITLE
Added interface and changes for blx config file

### PR DESCRIPTION
**Pull request by akashram14**
Can someone check if this is right?

### CODE REVIEW

File: sample.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  remote_user: root
+  vars:
+          main_iface_name: enp0s25
+          work_process: 2
+          ip_addr: w.x.y.z
+          gateway: w.x.y.z
+
+  tasks:
+        - name: Creating new macvlan interface
+          command: ip link add newiface link {{ main_iface_name  }} type macvlan mode bridge
```
Comment (oppiliappan) [inline sample.yml:11]: Let us not hardcode the interface name:
```
  vars:
    iface: "enp3s0"    # will be provided via Nitin's shell script
    sub_iface: sub0{{ iface }}    # subinterface name adds a prefix to existing interface
```

Set interface state to UP and disable GRO; GSO as well:

```
  tasks:
    - name: Create new sub-interface
      become: yes
      register: sub_iface_result # we should use this in the next play
      shell: 
        cmd: ip link add {{ sub_iface }} link {{ iface  }} type macvlan mode bridge

    - name: Set interface state to up
      become: yes
      shell:
        cmd: ifconfig {{ sub_iface }} up

    - name: Set GRO and GSO to OFF
      become: yes
      shell:
        cmd: sudo ethtool -K {{ sub_iface }} gro off gso off

```

Comment (akashram14) [reply sample.yml:11]: KK but anyways Rohit is using another method to change the config file i think he is using variable for the sub-interface name

File: sample.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        - name: Changing the config file.
+          lineinfile:
+                  path: /etc/blx/blx.conf
+                  regexp: '{{item.From}}'
+                  line: '{{item.To}}'
+                  state: present
+          with_items:
+                  - { From: '# worker-processes:5', To: 'worker-processes:{{work_process}}'}
+                  - { From: '# ipaddress: 7.7.7.10/24', To: 'ipaddress: {{ip_addr}}'}
+                  - { From: '# interfaces: sub_interface_name:peth1,type:macvlan,interface_name:eth1', To: ' interfaces: sub_interface_name:newiface,type:macvlan,interface_name:{{main_iface_name}}' }
```
Comment (oppiliappan) [inline sample.yml:22]: Same as previous, use interface name from variable, looks good otherwise.

Comment (akashram14) [reply sample.yml:22]: Yea this part(changing the config file) Rohit used a different way using some template i think.

Comment (oppiliappan) [reply sample.yml:22]: Right, I think a template is better, let us not use this regex play.

Comment (akashram14) [reply sample.yml:22]: Ok. So shall i just close this pull request?

Comment (oppiliappan) [reply sample.yml:22]: Yeah, I have made some of the interface related changes, and integrated Nitin's templating changes.

**Pull request closed without merging**